docs/source/getting-started/registering-first-model/step3-load-model.rst

```diff
@@ -0,0 +1,84 @@
+Load a Registered Model
+=======================
+
+To perform inference on a registered model version, we need to load it into memory. There are many 
+ways to find our model version, but the best method differs depending on the information you have
+available. However, in the spirit of a quickstart, here is the method that requires the least amount
+of information.
+
+.. code-block:: python
+
+    import mlflow.sklearn
+    from sklearn.datasets import make_regression
+
+    model_name = "sk-learn-random-forest-reg-model"
+    model_version = "latest"
+
+    # Load the model from the Model Registry
+    model_uri = f"models:/{model_name}/{model_version}"
+    model = mlflow.sklearn.load_model(model_uri)
+
+    # Generate a new dataset for prediction and predict
+    X_new, _ = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+    y_pred_new = model.predict(X_new)
+
+    print(y_pred_new)
+
+
+Example 1: Load via Static Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are many other ways to construct a model URI. Here are some common examples that leverage 
+information created at registration time. 
+
+1. **Absolute local path**: ``mlflow.sklearn.load_model("/Users/me/path/to/local/model")``
+2. **Relative local path**: ``mlflow.sklearn.load_model("relative/path/to/local/model")``
+3. **Run id**: ``mlflow.sklearn.load_model(f"runs:/{mlflow_run_id}/{run_relative_path_to_model}")``
+4. **Model name + version**: ``mlflow.sklearn.load_model(f"models:/{model_name}/{model_version}")``
+
+After registration, you can also leverage additional metadata to facilitate finding the model.
+
+Example 2: Load via Model Version Alias
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Model version aliases are user-defined identifiers for a model version. Given they're mutable after
+model registration, they're more versatile than monotonically increasing ID's. In the prior page, we
+added a model version alias to our model, but here's a programmatic example.
```

Comment:
Nit on content: aliases are not really mutable in the sense that they can be renamed (you need to delete old one and create new oes), they're mutable because they can be reassigned via the UI or API. So if you had workloads pointing to a certain alias, you can update all your workloads to a new model version simply by reassigning the aliases. That's where the real power of aliases come in. I'm not sure that comes through at the moment.